team-config.yml

```
@@ -26,7 +26,7 @@ soc:
   team: "soc"
   namespace: ""
   slack:
-    contact_channel: "#dtspo-secops-chat"
+    contact_channel: "#devops-secops-chat"
```

shouldn't this just be
```suggestion
    contact_channel: "#sec-ops"
```